src/core/tests/Test_StatusHandler.py

```diff
@@ -396,6 +398,77 @@ def test_sort_packages_by_classification_and_state(self):
             self.assertEqual(installation_patches_sorted[12]["name"], "test-package-2")  # | Other              | Excluded    |
             self.assertEqual(installation_patches_sorted[13]["name"], "test-package-1")  # | Other              | NotSelected |
 
+    def test_if_complete_status_has_bad_json(self):
```

Comment:
This test is checking what happens when *.complete.status has an empty substatus. Empty substatus is not a bad json.
Please rename the function to test_complete_status_with_empty_substatus()